src/Collage/Sketchy.elm

```diff
@@ -0,0 +1,234 @@
+module Collage.Sketchy exposing (Config, defaultConfig, sketchy, nextSeed)
```

Comment:
This and `Sketchy.Fill` look good to me. I didn't look into the details, as I think you researched that yourself better then I did ;-)